### ISSUE TITLE
AWS S3 instead of Cloudinary

### ISSUE THREAD

**Issue by codenamethanos** [render-examples/strapi-postgres-cloudinary#14]
Is it possible that I can use AWS S3 instead of Cloudinary? 
Thanks

**Comment by davidmauskop** [comment]
Yes! I'd recommend that you fork the repo and then replace `strapi-provider-upload-cloudinary` with `strapi-provider-upload-aws-s3`. You'll want to update the `package.json` file, and also update the production config (https://github.com/render-examples/strapi-postgres-cloudinary/blob/master/config/env/production/plugins.js) based on the instructions at https://www.npmjs.com/package/strapi-provider-upload-aws-s3.
